File: sample-conference-videochat/sample-conference-videochat/Components/Settings/VideoSettingsViewController/SwitchItemModel.m
//
//  SwitchItemModel.m
//  sample-conference-videochat
//
//  Created by Injoit on 30/09/15.
//  Copyright © 2015 QuickBlox Team. All rights reserved.
//

#import "SwitchItemModel.h"
#import "SettingSwitchCell.h"

@implementation SwitchItemModel

- (Class)viewClass {
    
    return [SettingSwitchCell class];
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Components/Settings/SwitchItemModel.swift
//
//  SwitchItemModel.swift
//  sample-conference-videochat-swift
//
//  Created by Injoit on 22.10.2018.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import Foundation

class SwitchItemModel: BaseItemModel {
    //MARK: - Properties
    var on = false
    
    //MARK: - Overrides
    override func viewClass() -> AnyClass {
        return SettingSwitchCell.self
    }
}
